sheeprl/algos/ppo/args.py

```diff
@@ -32,4 +32,4 @@ class PPOArgs(StandardArgs):
     anneal_ent_coef: bool = Arg(default=False, help="whether to linearly anneal the entropy coefficient to zero")
     vf_coef: float = Arg(default=1.0, help="coefficient of the value function")
     max_grad_norm: float = Arg(default=0.0, help="the maximum norm for the gradient clipping")
-    checkpoint_every: int = Arg(default=10, help="how often to make the checkpoint, -1 to deactivate the checkpoint")
+    checkpoint_every: int = Arg(default=100000, help="how often to make the checkpoint, -1 to deactivate the checkpoint")
```

Comment:
For PPO this is a very huge number and we risk that no model is saved during the optimization. Can we maybe set the checkpoint_every as `Optional[int]` and `None` by default and if None we save only the last checkpoint? Or set it to -1 and apply the same reasoning?

Reply:
Initially I set it to -1, but I thought you wanted to keep it